README.md

```diff
@@ -8,16 +8,397 @@ so don't try to use it for anything close to a production environment.
+  metadataServerConfig:
+    enabled: true
+  resourceServerConfig:
+    enabled: false
+```
+Next, apply the sample ORANO2IMS custom resource with the new configuration. 
+
+```bash
+$ oc apply -f sample-oran2ims-dms.yaml 
+orano2ims.oran.openshift.io/sample configured
+```
+
+See that a new Pod started in the oran-o2ims namespace.
+
+```bash
+$ oc get pods -n oran-o2ims
+NAME                                         READY   STATUS    RESTARTS   AGE
+deployment-manager-server-785997bd4b-r8qbt   1/1     Running   0          3s
+metadata-server-c655b559b-tf7f7              1/1     Running   0          13h
+```
+
+Check that the Pod started successfully, you can see the output of the Pod to look for errors or check the status of the ORAN2IMS custom resource:
+
+```bash
+$ oc get orano2imses.oran.openshift.io sample -ojsonpath={.status.deploymentStatus} | jq
+{
+  "conditions": [
+    {
+      "lastTransitionTime": "2024-07-24T07:02:48Z",
+      "message": "Deployment has minimum availability.",
+      "reason": "MinimumReplicasAvailable",
+      "status": "True",
+      "type": "MetadataServerAvailable"
+    },
+    {
+      "lastTransitionTime": "2024-07-24T07:17:48Z",
+      "message": "Deployment has minimum availability.",
+      "reason": "MinimumReplicasAvailable",
+      "status": "True",
+      "type": "DeploymentServerAvailable"
+    }
+  ]
+}
+```
+
+##### Requests Examples
+
+###### GET deploymentManagers List
+
+To get a list of deploymentManagers:
+
+```bash
+$ curl --insecure --silent --header "Authorization: Bearer $CLIENT_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/deploymentManagers" | jq
+```
+
+###### GET fields from the deploymentManagers List
+
+To get only the name of the available deploymentManagers:
+
+```bash
+$ curl --insecure --silent --header "Authorization: Bearer $CLIENT_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/deploymentManagers?fields=name" | jq
+```
+
+###### GET deploymentManagers List using filter
+
+To get the deploymentManagers whose name is **not** local-cluster.
+
+```bash
+$ curl --insecure --silent --header "Authorization: Bearer $CLIENT_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/deploymentManagers?filter=(neq,name,local-cluster)" | jq
+ | jq
+```
+
+####  1.2.3. <a name='Resourceserver'></a>Resource server
+
+The resource server exposes endpoints for retrieving resource types, resource pools
+and resources objects. The server relies on the Search Query API of ACM hub.
+Follow these [instructions](docs/dev/env_acm.md#search-query-api) to enable
+and configure the search API access. The resource server will translate those REST requests and send them to the ACM search server that implements a graphQL API.
+
+The required URL and token can be obtained as follows:
+
+```bash
+$ export BACKEND_URL=$(
+  oc get route -n open-cluster-management search-api -o json |
+  jq -r '"https://" + .spec.host'
+)
+$ export BACKEND_TOKEN_RS=$(
```

Comment:
Same comment as above regarding the backend token.  It is not necessary to specify one since the server will use its serviceaccount token automatically.